### DISCUSSION TITLE
RAPIDS - Data Analysis

### DISCUSSION THREAD

**Discussion by sureshksampath** [carissalow/rapids#218] [General]
Hello,

I have completed the configuration and execution steps; as a result, there are a few files generated in the processed feature directory. I am wondering what are the next steps. How do I perform data analysis and produce data visualization reports? Any pointers will be appreciated.

Thanks,
Suresh

**Comment by sureshksampath** [comment]
Also, could you please share any documentation around algorithms used to create processed and cleansed files?

**Reply by jenniferfedor** [reply]
Hi @sureshksampath,

Although we hope to add them in the future, there are currently no data analysis modules implemented in RAPIDS. However, the [complete workflow example](https://www.rapids.science/1.9/analysis/complete-workflow-example/) in our documentation provides an overview of the steps required to integrate your analysis into the RAPIDS framework.  

The Visualizations section of our documentation describes the plots of [data quality](https://www.rapids.science/1.9/visualizations/data-quality-visualizations/) and [processed sensor features](https://www.rapids.science/1.9/visualizations/feature-visualizations/) that are currently available in RAPIDS. To produce some or all of these reports, you can set the corresponding `PLOT` flag(s) in your [config file](https://github.com/carissalow/rapids/blob/a353e58c569f7b9515900337acf80cfcb1e2f7f3/config.yaml#L537) to `True`. You can also feel free to implement your own visualizations within or outside of RAPIDS. 

Finally, please feel free to refer to the Behavioral Features and Data Cleaning sections of our [documentation](https://www.rapids.science/1.9/) and the [`src/features`](https://github.com/carissalow/rapids/tree/master/src/features) directory in this repository for any details about how features are computed and processed data are cleaned.
